setup.py

```diff
@@ -52,7 +52,7 @@ def remove_bdist_rpm_source_file():
 
 PACKAGE = {
     'name': 'vsc-base',
-    'version': '2.2.1',
+    'version': '2.2.2',
```

Comment:
we have no way of controlling that today. Also it does not pose that big of a problem/issue.